doc/source/learning/overview.rst

```diff
@@ -0,0 +1,38 @@
+Overview
+=====================
+
+This part of the documentation will help new developers to understand what is Airship and all its
+ecosystem of tools.
```

Comment:
That looks redundant with Airship above.

The goal of this document is to help people onboard with socok8s, which is the current stepping stone for the Cloud10 product.

Reply:
I agree